frontend/src/pages/Profile/index.tsx

```diff
@@ -0,0 +1,226 @@
+              fill="#E6E6E6"
+            />
+          </svg>
+        </ProfileImage>
+        <Typography
+          variant="h2"
+          fontSize={{xs: '1rem', sm: '1.25rem', md: '1.5rem'}}
+          fontWeight={500}
+          marginTop="1.5rem"
+          marginBottom="1rem"
+          color="black"
+        >
+          Profile Details
+        </Typography>
+        <Box>
+          <InputField variant="outlined" label="Enter your name" />
+          <InputField variant="outlined" label="Enter your Email ID" />
+        </Box>
+        <Typography
+          variant="body2"
+          color={theme.palette.secondary.light}
+          fontWeight={600}
+        >
+          Member Since, May 2022
+        </Typography>
+      </ProfileSection>
+      <EventBox>
+        <Box
+          display="flex"
+          justifyContent="start"
+          alignItems="center"
+          marginBottom="1rem"
+          gap="20px"
+          borderBottom="2px solid #e6e6e6"
+        >
+          <Typography
+            variant="h4"
+            fontWeight={600}
+            color={theme.palette.primary.main}
+            borderBottom={currentTab === 1 ? '4px solid' : 'none'}
+            borderColor={theme.palette.primary.main}
+            paddingBottom="0.5rem"
+            onClick={() => {
+              setCurrentTab(1);
+            }}
+          >
+            Upcoming
+          </Typography>
+          <Typography
+            variant="h4"
+            fontWeight={600}
+            color={theme.palette.primary.main}
```

Comment:
unselected tab should be black in color otherwise main color of palette according to the design. You can achieve this by condtionally rendering the color according to the `currentTab` state value.

Reply:
ok.. i will fix this issue with the suggested method